clients/src/main/java/org/apache/kafka/clients/admin/Admin.java

```diff
@@ -915,23 +916,56 @@ default ListConsumerGroupsResult listConsumerGroups() {
 
     /**
      * List the consumer group offsets available in the cluster.
+     * <p>
+     * @deprecated Since 3.3.
+     * Use {@link #listConsumerGroupOffsets(Map, ListConsumerGroupOffsetsOptions)}.
      *
      * @param options The options to use when listing the consumer group offsets.
      * @return The ListGroupOffsetsResult
      */
-    ListConsumerGroupOffsetsResult listConsumerGroupOffsets(String groupId, ListConsumerGroupOffsetsOptions options);
+    @Deprecated
```

Comment:
We had two methods earlier, 
- ListConsumerGroupOffsetsResult listConsumerGroupOffsets(String groupId)
- ListConsumerGroupOffsetsResult listConsumerGroupOffsets(String groupId, ListConsumerGroupOffsetsOptions options) - this one.

The PR was deprecating both earlier. I thought it was useful to retain the first one, so removed that deprecation. Not so sure about the second one because that relies on topic partition list in ListConsumerGroupOffsetsOptions, which I thought we should deprecate. Hence left this.

Reply:
I have reverted the deprecation of this method. Only the topicPartitions method in ListConsumerGroupOffsetsOptions have been deprecated, since there is an equivalent in ListConsumerGroupOffsetsSpec now.